### ISSUE TITLE
[bitnami/thanos]: Receiver can not run without object store config

### ISSUE THREAD

**Issue by badrabubker** [bitnami/charts#10995]
### Name and Version

bitnami/thanos 10.5.4

### What steps will reproduce the bug?

1. install Thanos helm chart
2. Enable Thanos receiver
3. see the validation error 
```
....
VALUES VALIDATION: thanos: objstore configuration When enabling Bucket Web, Compactor, Ruler, Store or Receive Gateway component, you must provide a valid objstore configuration. There are three alternatives to provide it: 1) Provide it using the 'objstoreConfig' parameter 2) Provide it using an existing Secret and using the 'existingObjstoreSecret' parameter 3) Put your objstore.yml under the 'files/conf/' directory Use --debug flag to render out invalid YAML [helm-sops] Error: failed to run Helm: exit status 1
```
that comes from https://github.com/bitnami/charts/blob/master/bitnami/thanos/templates/_helpers.tpl#L200-L209

### Are you using any custom parameters or values?

```
    receive:
      enabled: true
```

### What is the expected behavior?

thanos receiver can start without the object store config file

### What do you see instead?

validation error 

### Additional information

I am happy to submit a MR for that

**Comment by carrodher** [comment]
Currently, there is a validation checking that a proper object storage configuration is provided when the mentioned components are enabled. What's your use case? Why do you need to enable the receiver without object storage?

**Comment by badrabubker** [comment]
Right I found the validation in the mentioned link.
Currently I am running thanos receiver binary (without the helm chat) and I wanted to run the same with helm.

My use case is the following:
I have multiple Prometheus instances that don't need long term storage. These instances are deployed with thanos sidecare and there is a thanos querier and thanos receiver (acting as push entry point) 
The querier is acting as aggregator and a single point of querying. Thanos receiver can store date locally which makes me wonder why the validation was implemented from the beginning as Thanos receiver doesn't complain if you run it without object store config and I couldn't find anything in the documentation that discourages that.


Thanks :)
